ui/contextmenu.js

```
@@ -0,0 +1,857 @@
+  (function removeRedundantContextMenuItems() {
+    const registry = Blockly.ContextMenuRegistry.registry;
+    [
+      'undoWorkspace',
+      'redoWorkspace',
+      'cleanWorkspace',
+      'collapseWorkspace',
+      'expandWorkspace',
+    ].forEach((id) => {
+      try {
+        registry.unregister(id);
+      } catch (_) {}
```

ESLint

[error] '_' is defined but never used.

(no-unused-vars)